### ISSUE TITLE
Terminology - state vs journal

### ISSUE THREAD

**Issue by omesser** [iterative/gto#69]
tl;dr: State seems to be more like a journal or audit log with labels and versions actions being listed, it's not current state

UX - In something named `state` it's weird to have the behavior of repeating an action that makes no change (registering the same model over the same commit without a change) - and seeing new entries in the "state".
To reproduce simply re-run `gto promote [name] [label] --version [version]` repeatedly without any change

Feels like the `print-state` shows a de-facto journal / audit-log.
Related (UX 🐛 ?) -  for audit/log-like "expected behavior" - demoting / removing label should be a separate line. every action should be an item / row as is expected in audit logs, otherwise the users has to "compile" the sequence of events from the cli output in their heads

It would be nice to see some sort of actual "current" state - that would show the current label on each commit (and lose the history)

@aguschin  @dberenbaum  - thoughts?
